diff --git a/D7UPGRADE.txt b/D7UPGRADE.txt
old mode 100644
new mode 100755
diff --git a/README.txt b/README.txt
old mode 100644
new mode 100755
diff --git a/css/ie/views-admin.ie7.css b/css/ie/views-admin.ie7.css
old mode 100644
new mode 100755
diff --git a/css/views-admin-rtl.css b/css/views-admin-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.advanced_help.css b/css/views-admin.advanced_help.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.bartik-rtl.css b/css/views-admin.bartik-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.bartik.css b/css/views-admin.bartik.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.contextual.css b/css/views-admin.contextual.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.css b/css/views-admin.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.ctools-rtl.css b/css/views-admin.ctools-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.ctools.css b/css/views-admin.ctools.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.garland-rtl.css b/css/views-admin.garland-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.garland.css b/css/views-admin.garland.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.seven-rtl.css b/css/views-admin.seven-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.seven.css b/css/views-admin.seven.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.theme-rtl.css b/css/views-admin.theme-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views-admin.theme.css b/css/views-admin.theme.css
old mode 100644
new mode 100755
diff --git a/css/views-rtl.css b/css/views-rtl.css
old mode 100644
new mode 100755
diff --git a/css/views.css b/css/views.css
old mode 100644
new mode 100755
diff --git a/documentation-standards.txt b/documentation-standards.txt
old mode 100644
new mode 100755
diff --git a/drush/views.drush.inc b/drush/views.drush.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_area.inc b/handlers/views_handler_area.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_area_result.inc b/handlers/views_handler_area_result.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_area_text.inc b/handlers/views_handler_area_text.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_area_text_custom.inc b/handlers/views_handler_area_text_custom.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_area_view.inc b/handlers/views_handler_area_view.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument.inc b/handlers/views_handler_argument.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_date.inc b/handlers/views_handler_argument_date.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_formula.inc b/handlers/views_handler_argument_formula.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_group_by_numeric.inc b/handlers/views_handler_argument_group_by_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_many_to_one.inc b/handlers/views_handler_argument_many_to_one.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_null.inc b/handlers/views_handler_argument_null.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_numeric.inc b/handlers/views_handler_argument_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_argument_string.inc b/handlers/views_handler_argument_string.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field.inc b/handlers/views_handler_field.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_boolean.inc b/handlers/views_handler_field_boolean.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_contextual_links.inc b/handlers/views_handler_field_contextual_links.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_counter.inc b/handlers/views_handler_field_counter.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_custom.inc b/handlers/views_handler_field_custom.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_date.inc b/handlers/views_handler_field_date.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_entity.inc b/handlers/views_handler_field_entity.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_machine_name.inc b/handlers/views_handler_field_machine_name.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_markup.inc b/handlers/views_handler_field_markup.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_math.inc b/handlers/views_handler_field_math.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_numeric.inc b/handlers/views_handler_field_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_prerender_list.inc b/handlers/views_handler_field_prerender_list.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_serialized.inc b/handlers/views_handler_field_serialized.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_time_interval.inc b/handlers/views_handler_field_time_interval.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_field_url.inc b/handlers/views_handler_field_url.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter.inc b/handlers/views_handler_filter.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_boolean_operator.inc b/handlers/views_handler_filter_boolean_operator.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_boolean_operator_string.inc b/handlers/views_handler_filter_boolean_operator_string.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_combine.inc b/handlers/views_handler_filter_combine.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_date.inc b/handlers/views_handler_filter_date.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_entity_bundle.inc b/handlers/views_handler_filter_entity_bundle.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_equality.inc b/handlers/views_handler_filter_equality.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_group_by_numeric.inc b/handlers/views_handler_filter_group_by_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_in_operator.inc b/handlers/views_handler_filter_in_operator.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_many_to_one.inc b/handlers/views_handler_filter_many_to_one.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_numeric.inc b/handlers/views_handler_filter_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_filter_string.inc b/handlers/views_handler_filter_string.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_relationship.inc b/handlers/views_handler_relationship.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_relationship_groupwise_max.inc b/handlers/views_handler_relationship_groupwise_max.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_sort.inc b/handlers/views_handler_sort.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_sort_date.inc b/handlers/views_handler_sort_date.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_sort_group_by_numeric.inc b/handlers/views_handler_sort_group_by_numeric.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_sort_menu_hierarchy.inc b/handlers/views_handler_sort_menu_hierarchy.inc
old mode 100644
new mode 100755
diff --git a/handlers/views_handler_sort_random.inc b/handlers/views_handler_sort_random.inc
old mode 100644
new mode 100755
diff --git a/help/about.html b/help/about.html
old mode 100644
new mode 100755
diff --git a/help/advanced-settings.html b/help/advanced-settings.html
old mode 100644
new mode 100755
diff --git a/help/advanced-style-settings.html b/help/advanced-style-settings.html
old mode 100644
new mode 100755
diff --git a/help/aggregation.html b/help/aggregation.html
old mode 100644
new mode 100755
diff --git a/help/alter-exposed-filter.html b/help/alter-exposed-filter.html
old mode 100644
new mode 100755
diff --git a/help/analyze-theme.html b/help/analyze-theme.html
old mode 100644
new mode 100755
diff --git a/help/api-default-views.html b/help/api-default-views.html
old mode 100644
new mode 100755
diff --git a/help/api-example.html b/help/api-example.html
old mode 100644
new mode 100755
diff --git a/help/api-forms.html b/help/api-forms.html
old mode 100644
new mode 100755
diff --git a/help/api-handler-area.html b/help/api-handler-area.html
old mode 100644
new mode 100755
diff --git a/help/api-tables.html b/help/api-tables.html
old mode 100644
new mode 100755
diff --git a/help/api-upgrading.html b/help/api-upgrading.html
old mode 100644
new mode 100755
diff --git a/help/api.html b/help/api.html
old mode 100644
new mode 100755
diff --git a/help/argument.html b/help/argument.html
old mode 100644
new mode 100755
diff --git a/help/basic-settings.html b/help/basic-settings.html
old mode 100644
new mode 100755
diff --git a/help/demo-video.html b/help/demo-video.html
old mode 100644
new mode 100755
diff --git a/help/display-attachment.html b/help/display-attachment.html
old mode 100644
new mode 100755
diff --git a/help/display-block.html b/help/display-block.html
old mode 100644
new mode 100755
diff --git a/help/display-default.html b/help/display-default.html
old mode 100644
new mode 100755
diff --git a/help/display-feed.html b/help/display-feed.html
old mode 100644
new mode 100755
diff --git a/help/display-page.html b/help/display-page.html
old mode 100644
new mode 100755
diff --git a/help/display.html b/help/display.html
old mode 100644
new mode 100755
diff --git a/help/drush.html b/help/drush.html
old mode 100644
new mode 100755
diff --git a/help/embed.html b/help/embed.html
old mode 100644
new mode 100755
diff --git a/help/empty-text.html b/help/empty-text.html
old mode 100644
new mode 100755
diff --git a/help/example-author-block.html b/help/example-author-block.html
old mode 100644
new mode 100755
diff --git a/help/example-filter-by-current-user.html b/help/example-filter-by-current-user.html
old mode 100644
new mode 100755
diff --git a/help/example-recent-stories.html b/help/example-recent-stories.html
old mode 100644
new mode 100755
diff --git a/help/example-slideshow-thumb-pager.html b/help/example-slideshow-thumb-pager.html
old mode 100644
new mode 100755
diff --git a/help/example-user-feed.html b/help/example-user-feed.html
old mode 100644
new mode 100755
diff --git a/help/example-users-by-role.html b/help/example-users-by-role.html
old mode 100644
new mode 100755
diff --git a/help/exposed-form.html b/help/exposed-form.html
old mode 100644
new mode 100755
diff --git a/help/field.html b/help/field.html
old mode 100644
new mode 100755
diff --git a/help/filter.html b/help/filter.html
old mode 100644
new mode 100755
diff --git a/help/get-total-rows.html b/help/get-total-rows.html
old mode 100644
new mode 100755
diff --git a/help/getting-started.html b/help/getting-started.html
old mode 100644
new mode 100755
diff --git a/help/group-by.html b/help/group-by.html
old mode 100644
new mode 100755
diff --git a/help/header.html b/help/header.html
old mode 100644
new mode 100755
diff --git a/help/images/node-term_node-term_data-large.png b/help/images/node-term_node-term_data-large.png
old mode 100644
new mode 100755
diff --git a/help/images/node-term_node-term_data.png b/help/images/node-term_node-term_data.png
old mode 100644
new mode 100755
diff --git a/help/images/overview-ui-large.png b/help/images/overview-ui-large.png
old mode 100644
new mode 100755
diff --git a/help/images/overview-ui-small.png b/help/images/overview-ui-small.png
old mode 100644
new mode 100755
diff --git a/help/images/style-breakdown-large.png b/help/images/style-breakdown-large.png
old mode 100644
new mode 100755
diff --git a/help/images/style-breakdown.png b/help/images/style-breakdown.png
old mode 100644
new mode 100755
diff --git a/help/images/views1-admin-large.png b/help/images/views1-admin-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views1-admin.png b/help/images/views1-admin.png
old mode 100644
new mode 100755
diff --git a/help/images/views1-changeviewtype-large.png b/help/images/views1-changeviewtype-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views1-changeviewtype.png b/help/images/views1-changeviewtype.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addaview-large.png b/help/images/views2-addaview-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addaview.png b/help/images/views2-addaview.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-adddisplay-large.png b/help/images/views2-adddisplay-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-adddisplay.png b/help/images/views2-adddisplay.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addfields-large.png b/help/images/views2-addfields-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addfields.png b/help/images/views2-addfields.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addfieldsajax-large.png b/help/images/views2-addfieldsajax-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-addfieldsajax.png b/help/images/views2-addfieldsajax.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-admin-large.png b/help/images/views2-admin-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-admin.png b/help/images/views2-admin.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-changedisplaystyle-large.png b/help/images/views2-changedisplaystyle-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-changedisplaystyle.png b/help/images/views2-changedisplaystyle.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-fieldspreview-large.png b/help/images/views2-fieldspreview-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-fieldspreview.png b/help/images/views2-fieldspreview.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-newview-large.png b/help/images/views2-newview-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-newview.png b/help/images/views2-newview.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-rearrangefields-large.png b/help/images/views2-rearrangefields-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-rearrangefields.png b/help/images/views2-rearrangefields.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-tablestyle-large.png b/help/images/views2-tablestyle-large.png
old mode 100644
new mode 100755
diff --git a/help/images/views2-tablestyle.png b/help/images/views2-tablestyle.png
old mode 100644
new mode 100755
diff --git a/help/images/views3-group-aggregation-types.png b/help/images/views3-group-aggregation-types.png
old mode 100644
new mode 100755
diff --git a/help/images/views3-group-aggregation.png b/help/images/views3-group-aggregation.png
old mode 100644
new mode 100755
diff --git a/help/images/views3-jump-style-menu.png b/help/images/views3-jump-style-menu.png
old mode 100644
new mode 100755
diff --git a/help/images/views3-semanticviews.png b/help/images/views3-semanticviews.png
old mode 100644
new mode 100755
diff --git a/help/images/views3-views-all.png b/help/images/views3-views-all.png
old mode 100644
new mode 100755
diff --git a/help/menu.html b/help/menu.html
old mode 100644
new mode 100755
diff --git a/help/misc-notes.html b/help/misc-notes.html
old mode 100644
new mode 100755
diff --git a/help/new.html b/help/new.html
old mode 100644
new mode 100755
diff --git a/help/other-help.html b/help/other-help.html
old mode 100644
new mode 100755
diff --git a/help/overrides.html b/help/overrides.html
old mode 100644
new mode 100755
diff --git a/help/path.html b/help/path.html
old mode 100644
new mode 100755
diff --git a/help/performance-views-vs-displays.html b/help/performance-views-vs-displays.html
old mode 100644
new mode 100755
diff --git a/help/performance.html b/help/performance.html
old mode 100644
new mode 100755
diff --git a/help/relationship-representative.html b/help/relationship-representative.html
old mode 100644
new mode 100755
diff --git a/help/relationship.html b/help/relationship.html
old mode 100644
new mode 100755
diff --git a/help/reports.html b/help/reports.html
old mode 100644
new mode 100755
diff --git a/help/select-multple-nids-contextual-filters.html b/help/select-multple-nids-contextual-filters.html
old mode 100644
new mode 100755
diff --git a/help/semantic-views.html b/help/semantic-views.html
old mode 100644
new mode 100755
diff --git a/help/sort.html b/help/sort.html
old mode 100644
new mode 100755
diff --git a/help/style-comment-rss.html b/help/style-comment-rss.html
old mode 100644
new mode 100755
diff --git a/help/style-fields.html b/help/style-fields.html
old mode 100644
new mode 100755
diff --git a/help/style-grid.html b/help/style-grid.html
old mode 100644
new mode 100755
diff --git a/help/style-grouping.html b/help/style-grouping.html
old mode 100644
new mode 100755
diff --git a/help/style-jump.html b/help/style-jump.html
old mode 100644
new mode 100755
diff --git a/help/style-list.html b/help/style-list.html
old mode 100644
new mode 100755
diff --git a/help/style-node-rss.html b/help/style-node-rss.html
old mode 100644
new mode 100755
diff --git a/help/style-node.html b/help/style-node.html
old mode 100644
new mode 100755
diff --git a/help/style-row.html b/help/style-row.html
old mode 100644
new mode 100755
diff --git a/help/style-rss.html b/help/style-rss.html
old mode 100644
new mode 100755
diff --git a/help/style-settings.html b/help/style-settings.html
old mode 100644
new mode 100755
diff --git a/help/style-summary-unformatted.html b/help/style-summary-unformatted.html
old mode 100644
new mode 100755
diff --git a/help/style-summary.html b/help/style-summary.html
old mode 100644
new mode 100755
diff --git a/help/style-table.html b/help/style-table.html
old mode 100644
new mode 100755
diff --git a/help/style-unformatted.html b/help/style-unformatted.html
old mode 100644
new mode 100755
diff --git a/help/style.html b/help/style.html
old mode 100644
new mode 100755
diff --git a/help/taxonomy-page-override.html b/help/taxonomy-page-override.html
old mode 100644
new mode 100755
diff --git a/help/theme-css.html b/help/theme-css.html
old mode 100644
new mode 100755
diff --git a/help/top-pager.html b/help/top-pager.html
old mode 100644
new mode 100755
diff --git a/help/ui-crashes.html b/help/ui-crashes.html
old mode 100644
new mode 100755
diff --git a/help/updating-view3.html b/help/updating-view3.html
old mode 100644
new mode 100755
diff --git a/help/updating.html b/help/updating.html
old mode 100644
new mode 100755
diff --git a/help/upgrading.html b/help/upgrading.html
old mode 100644
new mode 100755
diff --git a/help/using-theme.html b/help/using-theme.html
old mode 100644
new mode 100755
diff --git a/help/view-add.html b/help/view-add.html
old mode 100644
new mode 100755
diff --git a/help/view-settings.html b/help/view-settings.html
old mode 100644
new mode 100755
diff --git a/help/view-type.html b/help/view-type.html
old mode 100644
new mode 100755
diff --git a/help/views.help.ini b/help/views.help.ini
old mode 100644
new mode 100755
diff --git a/images/arrow-active.png b/images/arrow-active.png
old mode 100644
new mode 100755
diff --git a/images/close.png b/images/close.png
old mode 100644
new mode 100755
diff --git a/images/expanded-options.png b/images/expanded-options.png
old mode 100644
new mode 100755
diff --git a/images/loading-small.gif b/images/loading-small.gif
old mode 100644
new mode 100755
diff --git a/images/loading.gif b/images/loading.gif
old mode 100644
new mode 100755
diff --git a/images/overridden.gif b/images/overridden.gif
old mode 100644
new mode 100755
diff --git a/images/sprites.png b/images/sprites.png
old mode 100644
new mode 100755
diff --git a/images/status-active.gif b/images/status-active.gif
old mode 100644
new mode 100755
diff --git a/includes/admin.inc b/includes/admin.inc
old mode 100644
new mode 100755
diff --git a/includes/ajax.inc b/includes/ajax.inc
old mode 100644
new mode 100755
diff --git a/includes/analyze.inc b/includes/analyze.inc
old mode 100644
new mode 100755
diff --git a/includes/base.inc b/includes/base.inc
old mode 100644
new mode 100755
diff --git a/includes/cache.inc b/includes/cache.inc
old mode 100644
new mode 100755
diff --git a/includes/handlers.inc b/includes/handlers.inc
old mode 100644
new mode 100755
diff --git a/includes/plugins.inc b/includes/plugins.inc
old mode 100644
new mode 100755
diff --git a/includes/view.inc b/includes/view.inc
old mode 100644
new mode 100755
diff --git a/js/ajax.js b/js/ajax.js
old mode 100644
new mode 100755
diff --git a/js/ajax_view.js b/js/ajax_view.js
old mode 100644
new mode 100755
diff --git a/js/base.js b/js/base.js
old mode 100644
new mode 100755
diff --git a/js/jquery.ui.dialog.patch.js b/js/jquery.ui.dialog.patch.js
old mode 100644
new mode 100755
diff --git a/js/views-admin.js b/js/views-admin.js
old mode 100644
new mode 100755
diff --git a/js/views-contextual.js b/js/views-contextual.js
old mode 100644
new mode 100755
diff --git a/js/views-list.js b/js/views-list.js
old mode 100644
new mode 100755
diff --git a/modules/aggregator.views.inc b/modules/aggregator.views.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_argument_aggregator_category_cid.inc b/modules/aggregator/views_handler_argument_aggregator_category_cid.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_argument_aggregator_fid.inc b/modules/aggregator/views_handler_argument_aggregator_fid.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_argument_aggregator_iid.inc b/modules/aggregator/views_handler_argument_aggregator_iid.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_field_aggregator_category.inc b/modules/aggregator/views_handler_field_aggregator_category.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_field_aggregator_title_link.inc b/modules/aggregator/views_handler_field_aggregator_title_link.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_field_aggregator_xss.inc b/modules/aggregator/views_handler_field_aggregator_xss.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_handler_filter_aggregator_category_cid.inc b/modules/aggregator/views_handler_filter_aggregator_category_cid.inc
old mode 100644
new mode 100755
diff --git a/modules/aggregator/views_plugin_row_aggregator_rss.inc b/modules/aggregator/views_plugin_row_aggregator_rss.inc
old mode 100644
new mode 100755
diff --git a/modules/book.views.inc b/modules/book.views.inc
old mode 100644
new mode 100755
diff --git a/modules/book/views_plugin_argument_default_book_root.inc b/modules/book/views_plugin_argument_default_book_root.inc
old mode 100644
new mode 100755
diff --git a/modules/comment.views.inc b/modules/comment.views.inc
old mode 100644
new mode 100755
diff --git a/modules/comment.views_default.inc b/modules/comment.views_default.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_argument_comment_user_uid.inc b/modules/comment/views_handler_argument_comment_user_uid.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment.inc b/modules/comment/views_handler_field_comment.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_depth.inc b/modules/comment/views_handler_field_comment_depth.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_link.inc b/modules/comment/views_handler_field_comment_link.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_link_approve.inc b/modules/comment/views_handler_field_comment_link_approve.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_link_delete.inc b/modules/comment/views_handler_field_comment_link_delete.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_link_edit.inc b/modules/comment/views_handler_field_comment_link_edit.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_link_reply.inc b/modules/comment/views_handler_field_comment_link_reply.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_node_link.inc b/modules/comment/views_handler_field_comment_node_link.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_comment_username.inc b/modules/comment/views_handler_field_comment_username.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_last_comment_timestamp.inc b/modules/comment/views_handler_field_last_comment_timestamp.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_ncs_last_comment_name.inc b/modules/comment/views_handler_field_ncs_last_comment_name.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_ncs_last_updated.inc b/modules/comment/views_handler_field_ncs_last_updated.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_node_comment.inc b/modules/comment/views_handler_field_node_comment.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_field_node_new_comments.inc b/modules/comment/views_handler_field_node_new_comments.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_filter_comment_user_uid.inc b/modules/comment/views_handler_filter_comment_user_uid.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_filter_ncs_last_updated.inc b/modules/comment/views_handler_filter_ncs_last_updated.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_filter_node_comment.inc b/modules/comment/views_handler_filter_node_comment.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_sort_comment_thread.inc b/modules/comment/views_handler_sort_comment_thread.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_sort_ncs_last_comment_name.inc b/modules/comment/views_handler_sort_ncs_last_comment_name.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_handler_sort_ncs_last_updated.inc b/modules/comment/views_handler_sort_ncs_last_updated.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_plugin_row_comment_rss.inc b/modules/comment/views_plugin_row_comment_rss.inc
old mode 100644
new mode 100755
diff --git a/modules/comment/views_plugin_row_comment_view.inc b/modules/comment/views_plugin_row_comment_view.inc
old mode 100644
new mode 100755
diff --git a/modules/contact.views.inc b/modules/contact.views.inc
old mode 100644
new mode 100755
diff --git a/modules/contact/views_handler_field_contact_link.inc b/modules/contact/views_handler_field_contact_link.inc
old mode 100644
new mode 100755
diff --git a/modules/field.views.inc b/modules/field.views.inc
old mode 100644
new mode 100755
diff --git a/modules/field/views_handler_argument_field_list.inc b/modules/field/views_handler_argument_field_list.inc
old mode 100644
new mode 100755
diff --git a/modules/field/views_handler_argument_field_list_string.inc b/modules/field/views_handler_argument_field_list_string.inc
old mode 100644
new mode 100755
diff --git a/modules/field/views_handler_field_field.inc b/modules/field/views_handler_field_field.inc
old mode 100644
new mode 100755
diff --git a/modules/field/views_handler_filter_field_list.inc b/modules/field/views_handler_filter_field_list.inc
old mode 100644
new mode 100755
diff --git a/modules/field/views_handler_relationship_entity_reverse.inc b/modules/field/views_handler_relationship_entity_reverse.inc
old mode 100644
new mode 100755
diff --git a/modules/file.views.inc b/modules/file.views.inc
old mode 100644
new mode 100755
diff --git a/modules/filter.views.inc b/modules/filter.views.inc
old mode 100644
new mode 100755
diff --git a/modules/filter/views_handler_field_filter_format_name.inc b/modules/filter/views_handler_field_filter_format_name.inc
old mode 100644
new mode 100755
diff --git a/modules/image.views.inc b/modules/image.views.inc
old mode 100644
new mode 100755
diff --git a/modules/locale.views.inc b/modules/locale.views.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_argument_locale_group.inc b/modules/locale/views_handler_argument_locale_group.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_argument_locale_language.inc b/modules/locale/views_handler_argument_locale_language.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_field_locale_group.inc b/modules/locale/views_handler_field_locale_group.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_field_locale_language.inc b/modules/locale/views_handler_field_locale_language.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_field_locale_link_edit.inc b/modules/locale/views_handler_field_locale_link_edit.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_field_node_language.inc b/modules/locale/views_handler_field_node_language.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_filter_locale_group.inc b/modules/locale/views_handler_filter_locale_group.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_filter_locale_language.inc b/modules/locale/views_handler_filter_locale_language.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_filter_locale_version.inc b/modules/locale/views_handler_filter_locale_version.inc
old mode 100644
new mode 100755
diff --git a/modules/locale/views_handler_filter_node_language.inc b/modules/locale/views_handler_filter_node_language.inc
old mode 100644
new mode 100755
diff --git a/modules/node.views.inc b/modules/node.views.inc
old mode 100644
new mode 100755
diff --git a/modules/node.views_default.inc b/modules/node.views_default.inc
old mode 100644
new mode 100755
diff --git a/modules/node.views_template.inc b/modules/node.views_template.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_dates_various.inc b/modules/node/views_handler_argument_dates_various.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_node_language.inc b/modules/node/views_handler_argument_node_language.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_node_nid.inc b/modules/node/views_handler_argument_node_nid.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_node_type.inc b/modules/node/views_handler_argument_node_type.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_node_uid_revision.inc b/modules/node/views_handler_argument_node_uid_revision.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_argument_node_vid.inc b/modules/node/views_handler_argument_node_vid.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_history_user_timestamp.inc b/modules/node/views_handler_field_history_user_timestamp.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node.inc b/modules/node/views_handler_field_node.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_link.inc b/modules/node/views_handler_field_node_link.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_link_delete.inc b/modules/node/views_handler_field_node_link_delete.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_link_edit.inc b/modules/node/views_handler_field_node_link_edit.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_path.inc b/modules/node/views_handler_field_node_path.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_revision.inc b/modules/node/views_handler_field_node_revision.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_revision_link.inc b/modules/node/views_handler_field_node_revision_link.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_revision_link_delete.inc b/modules/node/views_handler_field_node_revision_link_delete.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_revision_link_revert.inc b/modules/node/views_handler_field_node_revision_link_revert.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_field_node_type.inc b/modules/node/views_handler_field_node_type.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_filter_history_user_timestamp.inc b/modules/node/views_handler_filter_history_user_timestamp.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_filter_node_access.inc b/modules/node/views_handler_filter_node_access.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_filter_node_status.inc b/modules/node/views_handler_filter_node_status.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_filter_node_type.inc b/modules/node/views_handler_filter_node_type.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_handler_filter_node_uid_revision.inc b/modules/node/views_handler_filter_node_uid_revision.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_plugin_argument_default_node.inc b/modules/node/views_plugin_argument_default_node.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_plugin_argument_validate_node.inc b/modules/node/views_plugin_argument_validate_node.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_plugin_row_node_rss.inc b/modules/node/views_plugin_row_node_rss.inc
old mode 100644
new mode 100755
diff --git a/modules/node/views_plugin_row_node_view.inc b/modules/node/views_plugin_row_node_view.inc
old mode 100644
new mode 100755
diff --git a/modules/poll.views.inc b/modules/poll.views.inc
old mode 100644
new mode 100755
diff --git a/modules/profile.views.inc b/modules/profile.views.inc
old mode 100644
new mode 100755
diff --git a/modules/profile/views_handler_field_profile_date.inc b/modules/profile/views_handler_field_profile_date.inc
old mode 100644
new mode 100755
diff --git a/modules/profile/views_handler_field_profile_list.inc b/modules/profile/views_handler_field_profile_list.inc
old mode 100644
new mode 100755
diff --git a/modules/profile/views_handler_filter_profile_selection.inc b/modules/profile/views_handler_filter_profile_selection.inc
old mode 100644
new mode 100755
diff --git a/modules/search.views.inc b/modules/search.views.inc
old mode 100644
new mode 100755
diff --git a/modules/search.views_default.inc b/modules/search.views_default.inc
old mode 100644
new mode 100755
diff --git a/modules/search/views_handler_argument_search.inc b/modules/search/views_handler_argument_search.inc
old mode 100644
new mode 100755
diff --git a/modules/search/views_handler_field_search_score.inc b/modules/search/views_handler_field_search_score.inc
old mode 100644
new mode 100755
diff --git a/modules/search/views_handler_filter_search.inc b/modules/search/views_handler_filter_search.inc
old mode 100644
new mode 100755
diff --git a/modules/search/views_handler_sort_search_score.inc b/modules/search/views_handler_sort_search_score.inc
old mode 100644
new mode 100755
diff --git a/modules/search/views_plugin_row_search_view.inc b/modules/search/views_plugin_row_search_view.inc
old mode 100644
new mode 100755
diff --git a/modules/statistics.views.inc b/modules/statistics.views.inc
old mode 100644
new mode 100755
diff --git a/modules/statistics.views_default.inc b/modules/statistics.views_default.inc
old mode 100644
new mode 100755
diff --git a/modules/statistics/views_handler_field_accesslog_path.inc b/modules/statistics/views_handler_field_accesslog_path.inc
old mode 100644
new mode 100755
diff --git a/modules/system.views.inc b/modules/system.views.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_argument_file_fid.inc b/modules/system/views_handler_argument_file_fid.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_field_file.inc b/modules/system/views_handler_field_file.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_field_file_extension.inc b/modules/system/views_handler_field_file_extension.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_field_file_filemime.inc b/modules/system/views_handler_field_file_filemime.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_field_file_status.inc b/modules/system/views_handler_field_file_status.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_field_file_uri.inc b/modules/system/views_handler_field_file_uri.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_filter_file_status.inc b/modules/system/views_handler_filter_file_status.inc
old mode 100644
new mode 100755
diff --git a/modules/system/views_handler_filter_system_type.inc b/modules/system/views_handler_filter_system_type.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy.views.inc b/modules/taxonomy.views.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy.views_default.inc b/modules/taxonomy.views_default.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_taxonomy.inc b/modules/taxonomy/views_handler_argument_taxonomy.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_term_node_tid.inc b/modules/taxonomy/views_handler_argument_term_node_tid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_term_node_tid_depth.inc b/modules/taxonomy/views_handler_argument_term_node_tid_depth.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_term_node_tid_depth_modifier.inc b/modules/taxonomy/views_handler_argument_term_node_tid_depth_modifier.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_vocabulary_machine_name.inc b/modules/taxonomy/views_handler_argument_vocabulary_machine_name.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_argument_vocabulary_vid.inc b/modules/taxonomy/views_handler_argument_vocabulary_vid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_field_taxonomy.inc b/modules/taxonomy/views_handler_field_taxonomy.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_field_term_link_edit.inc b/modules/taxonomy/views_handler_field_term_link_edit.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_field_term_node_tid.inc b/modules/taxonomy/views_handler_field_term_node_tid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_filter_term_node_tid.inc b/modules/taxonomy/views_handler_filter_term_node_tid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_filter_term_node_tid_depth.inc b/modules/taxonomy/views_handler_filter_term_node_tid_depth.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_filter_vocabulary_machine_name.inc b/modules/taxonomy/views_handler_filter_vocabulary_machine_name.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_filter_vocabulary_vid.inc b/modules/taxonomy/views_handler_filter_vocabulary_vid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_handler_relationship_node_term_data.inc b/modules/taxonomy/views_handler_relationship_node_term_data.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_plugin_argument_default_taxonomy_tid.inc b/modules/taxonomy/views_plugin_argument_default_taxonomy_tid.inc
old mode 100644
new mode 100755
diff --git a/modules/taxonomy/views_plugin_argument_validate_taxonomy_term.inc b/modules/taxonomy/views_plugin_argument_validate_taxonomy_term.inc
old mode 100644
new mode 100755
diff --git a/modules/translation.views.inc b/modules/translation.views.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_argument_node_tnid.inc b/modules/translation/views_handler_argument_node_tnid.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_field_node_link_translate.inc b/modules/translation/views_handler_field_node_link_translate.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_field_node_translation_link.inc b/modules/translation/views_handler_field_node_translation_link.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_filter_node_tnid.inc b/modules/translation/views_handler_filter_node_tnid.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_filter_node_tnid_child.inc b/modules/translation/views_handler_filter_node_tnid_child.inc
old mode 100644
new mode 100755
diff --git a/modules/translation/views_handler_relationship_translation.inc b/modules/translation/views_handler_relationship_translation.inc
old mode 100644
new mode 100755
diff --git a/modules/user.views.inc b/modules/user.views.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_argument_user_uid.inc b/modules/user/views_handler_argument_user_uid.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_argument_users_roles_rid.inc b/modules/user/views_handler_argument_users_roles_rid.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user.inc b/modules/user/views_handler_field_user.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_language.inc b/modules/user/views_handler_field_user_language.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_link.inc b/modules/user/views_handler_field_user_link.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_link_cancel.inc b/modules/user/views_handler_field_user_link_cancel.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_link_edit.inc b/modules/user/views_handler_field_user_link_edit.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_mail.inc b/modules/user/views_handler_field_user_mail.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_name.inc b/modules/user/views_handler_field_user_name.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_permissions.inc b/modules/user/views_handler_field_user_permissions.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_picture.inc b/modules/user/views_handler_field_user_picture.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_field_user_roles.inc b/modules/user/views_handler_field_user_roles.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_filter_user_current.inc b/modules/user/views_handler_filter_user_current.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_filter_user_name.inc b/modules/user/views_handler_filter_user_name.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_filter_user_permissions.inc b/modules/user/views_handler_filter_user_permissions.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_handler_filter_user_roles.inc b/modules/user/views_handler_filter_user_roles.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_plugin_argument_default_current_user.inc b/modules/user/views_plugin_argument_default_current_user.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_plugin_argument_default_user.inc b/modules/user/views_plugin_argument_default_user.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_plugin_argument_validate_user.inc b/modules/user/views_plugin_argument_validate_user.inc
old mode 100644
new mode 100755
diff --git a/modules/user/views_plugin_row_user_view.inc b/modules/user/views_plugin_row_user_view.inc
old mode 100644
new mode 100755
diff --git a/modules/views.views.inc b/modules/views.views.inc
old mode 100644
new mode 100755
diff --git a/plugins/export_ui/views_ui.class.php b/plugins/export_ui/views_ui.class.php
old mode 100644
new mode 100755
diff --git a/plugins/export_ui/views_ui.inc b/plugins/export_ui/views_ui.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_access.inc b/plugins/views_plugin_access.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_access_none.inc b/plugins/views_plugin_access_none.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_access_perm.inc b/plugins/views_plugin_access_perm.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_access_role.inc b/plugins/views_plugin_access_role.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_default.inc b/plugins/views_plugin_argument_default.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_default_fixed.inc b/plugins/views_plugin_argument_default_fixed.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_default_php.inc b/plugins/views_plugin_argument_default_php.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_default_raw.inc b/plugins/views_plugin_argument_default_raw.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_validate.inc b/plugins/views_plugin_argument_validate.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_validate_numeric.inc b/plugins/views_plugin_argument_validate_numeric.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_argument_validate_php.inc b/plugins/views_plugin_argument_validate_php.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_cache.inc b/plugins/views_plugin_cache.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_cache_none.inc b/plugins/views_plugin_cache_none.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_cache_time.inc b/plugins/views_plugin_cache_time.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display.inc b/plugins/views_plugin_display.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_attachment.inc b/plugins/views_plugin_display_attachment.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_block.inc b/plugins/views_plugin_display_block.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_default.inc b/plugins/views_plugin_display_default.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_embed.inc b/plugins/views_plugin_display_embed.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_extender.inc b/plugins/views_plugin_display_extender.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_feed.inc b/plugins/views_plugin_display_feed.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_display_page.inc b/plugins/views_plugin_display_page.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_exposed_form.inc b/plugins/views_plugin_exposed_form.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_exposed_form_basic.inc b/plugins/views_plugin_exposed_form_basic.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_exposed_form_input_required.inc b/plugins/views_plugin_exposed_form_input_required.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_localization.inc b/plugins/views_plugin_localization.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_localization_core.inc b/plugins/views_plugin_localization_core.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_localization_none.inc b/plugins/views_plugin_localization_none.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_pager.inc b/plugins/views_plugin_pager.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_pager_full.inc b/plugins/views_plugin_pager_full.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_pager_mini.inc b/plugins/views_plugin_pager_mini.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_pager_none.inc b/plugins/views_plugin_pager_none.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_pager_some.inc b/plugins/views_plugin_pager_some.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_query.inc b/plugins/views_plugin_query.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_query_default.inc b/plugins/views_plugin_query_default.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_row.inc b/plugins/views_plugin_row.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_row_fields.inc b/plugins/views_plugin_row_fields.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_row_rss_fields.inc b/plugins/views_plugin_row_rss_fields.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style.inc b/plugins/views_plugin_style.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_default.inc b/plugins/views_plugin_style_default.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_grid.inc b/plugins/views_plugin_style_grid.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_jump_menu.inc b/plugins/views_plugin_style_jump_menu.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_list.inc b/plugins/views_plugin_style_list.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_mapping.inc b/plugins/views_plugin_style_mapping.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_rss.inc b/plugins/views_plugin_style_rss.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_summary.inc b/plugins/views_plugin_style_summary.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_summary_jump_menu.inc b/plugins/views_plugin_style_summary_jump_menu.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_summary_unformatted.inc b/plugins/views_plugin_style_summary_unformatted.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_plugin_style_table.inc b/plugins/views_plugin_style_table.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/comment.inc b/plugins/views_wizard/comment.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/file_managed.inc b/plugins/views_wizard/file_managed.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/node.inc b/plugins/views_wizard/node.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/node_revision.inc b/plugins/views_wizard/node_revision.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/taxonomy_term.inc b/plugins/views_wizard/taxonomy_term.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/users.inc b/plugins/views_wizard/users.inc
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_base_views_wizard.class.php b/plugins/views_wizard/views_ui_base_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_comment_views_wizard.class.php b/plugins/views_wizard/views_ui_comment_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_file_managed_views_wizard.class.php b/plugins/views_wizard/views_ui_file_managed_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_node_revision_views_wizard.class.php b/plugins/views_wizard/views_ui_node_revision_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_node_views_wizard.class.php b/plugins/views_wizard/views_ui_node_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_taxonomy_term_views_wizard.class.php b/plugins/views_wizard/views_ui_taxonomy_term_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/plugins/views_wizard/views_ui_users_views_wizard.class.php b/plugins/views_wizard/views_ui_users_views_wizard.class.php
old mode 100644
new mode 100755
diff --git a/tests/comment/views_handler_argument_comment_user_uid.test b/tests/comment/views_handler_argument_comment_user_uid.test
old mode 100644
new mode 100755
diff --git a/tests/comment/views_handler_filter_comment_user_uid.test b/tests/comment/views_handler_filter_comment_user_uid.test
old mode 100644
new mode 100755
diff --git a/tests/field/views_fieldapi.test b/tests/field/views_fieldapi.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_area_text.test b/tests/handlers/views_handler_area_text.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_argument_null.test b/tests/handlers/views_handler_argument_null.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_argument_string.test b/tests/handlers/views_handler_argument_string.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field.test b/tests/handlers/views_handler_field.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_boolean.test b/tests/handlers/views_handler_field_boolean.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_counter.test b/tests/handlers/views_handler_field_counter.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_custom.test b/tests/handlers/views_handler_field_custom.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_date.test b/tests/handlers/views_handler_field_date.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_file_size.test b/tests/handlers/views_handler_field_file_size.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_math.test b/tests/handlers/views_handler_field_math.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_url.test b/tests/handlers/views_handler_field_url.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_field_xss.test b/tests/handlers/views_handler_field_xss.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_combine.test b/tests/handlers/views_handler_filter_combine.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_date.test b/tests/handlers/views_handler_filter_date.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_equality.test b/tests/handlers/views_handler_filter_equality.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_in_operator.test b/tests/handlers/views_handler_filter_in_operator.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_numeric.test b/tests/handlers/views_handler_filter_numeric.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_filter_string.test b/tests/handlers/views_handler_filter_string.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_sort.test b/tests/handlers/views_handler_sort.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_sort_date.test b/tests/handlers/views_handler_sort_date.test
old mode 100644
new mode 100755
diff --git a/tests/handlers/views_handler_sort_random.test b/tests/handlers/views_handler_sort_random.test
old mode 100644
new mode 100755
diff --git a/tests/node/views_node_revision_relations.test b/tests/node/views_node_revision_relations.test
old mode 100644
new mode 100755
diff --git a/tests/plugins/views_plugin_display.test b/tests/plugins/views_plugin_display.test
old mode 100644
new mode 100755
diff --git a/tests/styles/views_plugin_style.test b/tests/styles/views_plugin_style.test
old mode 100644
new mode 100755
diff --git a/tests/styles/views_plugin_style_base.test b/tests/styles/views_plugin_style_base.test
old mode 100644
new mode 100755
diff --git a/tests/styles/views_plugin_style_jump_menu.test b/tests/styles/views_plugin_style_jump_menu.test
old mode 100644
new mode 100755
diff --git a/tests/styles/views_plugin_style_mapping.test b/tests/styles/views_plugin_style_mapping.test
old mode 100644
new mode 100755
diff --git a/tests/styles/views_plugin_style_unformatted.test b/tests/styles/views_plugin_style_unformatted.test
old mode 100644
new mode 100755
diff --git a/tests/taxonomy/views_handler_relationship_node_term_data.test b/tests/taxonomy/views_handler_relationship_node_term_data.test
old mode 100644
new mode 100755
diff --git a/tests/templates/views-view--frontpage.tpl.php b/tests/templates/views-view--frontpage.tpl.php
old mode 100644
new mode 100755
diff --git a/tests/test_plugins/views_test_plugin_access_test_dynamic.inc b/tests/test_plugins/views_test_plugin_access_test_dynamic.inc
old mode 100644
new mode 100755
diff --git a/tests/test_plugins/views_test_plugin_access_test_static.inc b/tests/test_plugins/views_test_plugin_access_test_static.inc
old mode 100644
new mode 100755
diff --git a/tests/test_plugins/views_test_plugin_style_test_mapping.inc b/tests/test_plugins/views_test_plugin_style_test_mapping.inc
old mode 100644
new mode 100755
diff --git a/tests/user/views_handler_field_user_name.test b/tests/user/views_handler_field_user_name.test
old mode 100644
new mode 100755
diff --git a/tests/user/views_user.test b/tests/user/views_user.test
old mode 100644
new mode 100755
diff --git a/tests/user/views_user_argument_default.test b/tests/user/views_user_argument_default.test
old mode 100644
new mode 100755
diff --git a/tests/user/views_user_argument_validate.test b/tests/user/views_user_argument_validate.test
old mode 100644
new mode 100755
diff --git a/tests/views_access.test b/tests/views_access.test
old mode 100644
new mode 100755
diff --git a/tests/views_analyze.test b/tests/views_analyze.test
old mode 100644
new mode 100755
diff --git a/tests/views_argument_default.test b/tests/views_argument_default.test
old mode 100644
new mode 100755
diff --git a/tests/views_argument_validator.test b/tests/views_argument_validator.test
old mode 100644
new mode 100755
diff --git a/tests/views_basic.test b/tests/views_basic.test
old mode 100644
new mode 100755
diff --git a/tests/views_cache.test b/tests/views_cache.test
old mode 100644
new mode 100755
diff --git a/tests/views_cache.test.css b/tests/views_cache.test.css
old mode 100644
new mode 100755
diff --git a/tests/views_cache.test.js b/tests/views_cache.test.js
old mode 100644
new mode 100755
diff --git a/tests/views_exposed_form.test b/tests/views_exposed_form.test
old mode 100644
new mode 100755
diff --git a/tests/views_glossary.test b/tests/views_glossary.test
old mode 100644
new mode 100755
diff --git a/tests/views_groupby.test b/tests/views_groupby.test
old mode 100644
new mode 100755
diff --git a/tests/views_handlers.test b/tests/views_handlers.test
old mode 100644
new mode 100755
diff --git a/tests/views_module.test b/tests/views_module.test
old mode 100644
new mode 100755
diff --git a/tests/views_pager.test b/tests/views_pager.test
old mode 100644
new mode 100755
diff --git a/tests/views_plugin_localization_test.inc b/tests/views_plugin_localization_test.inc
old mode 100644
new mode 100755
diff --git a/tests/views_query.test b/tests/views_query.test
old mode 100644
new mode 100755
diff --git a/tests/views_test.info b/tests/views_test.info
old mode 100644
new mode 100755
diff --git a/tests/views_test.install b/tests/views_test.install
old mode 100644
new mode 100755
diff --git a/tests/views_test.module b/tests/views_test.module
old mode 100644
new mode 100755
diff --git a/tests/views_test.views_default.inc b/tests/views_test.views_default.inc
old mode 100644
new mode 100755
diff --git a/tests/views_translatable.test b/tests/views_translatable.test
old mode 100644
new mode 100755
diff --git a/tests/views_ui.test b/tests/views_ui.test
old mode 100644
new mode 100755
diff --git a/tests/views_upgrade.test b/tests/views_upgrade.test
old mode 100644
new mode 100755
diff --git a/tests/views_view.test b/tests/views_view.test
old mode 100644
new mode 100755
diff --git a/theme/theme.inc b/theme/theme.inc
old mode 100644
new mode 100755
diff --git a/theme/views-exposed-form.tpl.php b/theme/views-exposed-form.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-more.tpl.php b/theme/views-more.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-ui-display-tab-bucket.tpl.php b/theme/views-ui-display-tab-bucket.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-ui-display-tab-setting.tpl.php b/theme/views-ui-display-tab-setting.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-ui-edit-item.tpl.php b/theme/views-ui-edit-item.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-ui-edit-view.tpl.php b/theme/views-ui-edit-view.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-field.tpl.php b/theme/views-view-field.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-fields.tpl.php b/theme/views-view-fields.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-grid.tpl.php b/theme/views-view-grid.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-grouping.tpl.php b/theme/views-view-grouping.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-list.tpl.php b/theme/views-view-list.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-row-comment.tpl.php b/theme/views-view-row-comment.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-row-rss.tpl.php b/theme/views-view-row-rss.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-rss.tpl.php b/theme/views-view-rss.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-summary-unformatted.tpl.php b/theme/views-view-summary-unformatted.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-summary.tpl.php b/theme/views-view-summary.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-table.tpl.php b/theme/views-view-table.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view-unformatted.tpl.php b/theme/views-view-unformatted.tpl.php
old mode 100644
new mode 100755
diff --git a/theme/views-view.tpl.php b/theme/views-view.tpl.php
old mode 100644
new mode 100755
diff --git a/views.api.php b/views.api.php
old mode 100644
new mode 100755
diff --git a/views.info b/views.info
old mode 100644
new mode 100755
diff --git a/views.install b/views.install
old mode 100644
new mode 100755
diff --git a/views.module b/views.module
old mode 100644
new mode 100755
index 3b9eff1..35d10c5
--- a/views.module
+++ b/views.module
@@ -518,8 +518,10 @@ function views_arg_load($value, $name, $display_id, $index) {
 function views_page($name, $display_id) {
   $args = func_get_args();
   // Remove $name and $display_id from the arguments.
-  array_shift($args);
-  array_shift($args);
+  //this line changed to the previous implementation of views module by Stanislav Kutasevits (stanbellcom)
+  //to make it possible to use two contextual filters at the same time
+  //refer to http://www.scrumdo.com/projects/project/politiker-webapp/iteration/45959#story_319638 for more information
+  unset($args[0], $args[1]);
 
   // Load the view and render it.
   if ($view = views_get_view($name)) {
diff --git a/views.tokens.inc b/views.tokens.inc
old mode 100644
new mode 100755
diff --git a/views_export/views_export.module b/views_export/views_export.module
old mode 100644
new mode 100755
diff --git a/views_ui.info b/views_ui.info
old mode 100644
new mode 100755
diff --git a/views_ui.module b/views_ui.module
old mode 100644
new mode 100755
